jmp-cloud-service-impl/src/main/java/mzaba/subscription/rest/service/controller/ServiceControllerImpl.java

```diff
@@ -0,0 +1,81 @@
+package mzaba.subscription.rest.service.controller;
+
+import lombok.AllArgsConstructor;
+import mzaba.subscription.rest.domain.dto.SubscriptionRequestDto;
+import mzaba.subscription.rest.domain.dto.SubscriptionResponseDto;
+import mzaba.subscription.rest.service.SubscriptionService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/subscription-service/subscriptions")
+@AllArgsConstructor
+public class ServiceControllerImpl implements ServiceController{
+
+    private SubscriptionService subscriptionService;
+
+    @Override
+    @PostMapping
+    public ResponseEntity<SubscriptionResponseDto> createSubscription(@RequestBody SubscriptionRequestDto subscriptionRequest) {
+        var createdSubscription = subscriptionService.createSubscription(subscriptionRequest);
+        if(createdSubscription != null ){
+            return ResponseEntity.ok(createdSubscription);
+        } else {
+            return ResponseEntity.noContent().build();
```

Comment:
It's better to throw an error if the resource hasn't been created. With `noContent` it's unclear what has happened